_maps/map_files/Deltastation/DeltaStation2.dmm

```diff
@@ -43635,7 +43635,6 @@
 "kNB" = (
 /obj/structure/table/wood,
 /obj/item/papercutter,
-/obj/item/paper/fluff/ids_for_dummies,
```

Comment:
updatepaths script? 🥹

Reply:
I meaaaaan you can just find replace delete all..

Reply:
yeah but why would every downstream wanna do that on all their maps when you can just make a one line text file to automate it 🥹

Reply:
it would be ideal to have it for map depot and stuff, just
```txt
/obj/item/paper/fluff/ids_for_dummies : @DELETE
```